packages/network/src/NetworkNode.ts

```diff
@@ -101,4 +135,52 @@ export class NetworkNode extends Node {
         })
         return this.requestResend(request, null)
     }
+
+    /**
+     * Get number of neighbors for a given StreamPart
+     * @param streamId
+     * @param streamPartition
+     */
+    public getNumberOfNeighbors(streamId: string, streamPartition: number): number {
+        return this.getNeighborsFor(new StreamIdAndPartition(streamId, streamPartition)).length
+    }
+
+    /**
+     * Wait until at least the given number of neighbors are available on a stream.
+     * @param streamId
+     * @param streamPartition
+     * @param minNeighbors minimum number of neighbors to a wait for
+     * @param timeoutInMs timeout (in milliseconds) after which to give up waiting
+     * @returns nubmer of neighbors available
+     */
+    public waitForNeighbors(
```

Comment:
```suggestion
    public async waitForNeighbors(
```
Just for good measure. Everything is inside a new Promise so doesn't change anything in this case.

Reply:
committed.